makefile

```diff
@@ -109,6 +109,17 @@ pylint_py_files := \
     $(wildcard testsuite/test*.py) \
     testsuite/validate.py \
 
+# flake8 config file
+flake8_rc_file := flake8.rc
+
+# PyLint source files to check
+flake8_py_files := \
+    setup.py \
+    os_setup.py \
+    $(filter-out $(moftab_files), $(wildcard $(package_name)/*.py)) \
+    $(wildcard testsuite/test*.py) \
```

Comment:
Ouch. That is a direct copy from pylint_py_files so should change that one also or simply change it to line_py_files for both